components/batch-system/src/batch.rs

```diff
@@ -377,7 +397,8 @@ impl<N: Fsm, C: Fsm, Handler: PollHandler<N, C>> Poller<N, C, Handler> {
     }
 
     // Poll for readiness and forward to handler. Remove stale peer if necessary.
-    fn poll(&mut self) {
+    pub fn poll(&mut self) {
+        fail_point!("poll");
```

Comment:
What is this failpoint used for?

Reply:
> 

@5kbpers It's used for integration test:  https://github.com/tikv/tikv/pull/11213/files#diff-de27562fa1da856fe25ccd7973ce3ef87ff915dbd879db43deae2cf9ac39ea54R37.